test/parallel/test-promises-handled-reject.js

```diff
@@ -0,0 +1,17 @@
+// Flags: --expose-gc
+
+'use strict';
+const common = require('../common');
+
+const p = new Promise((res, rej) => {
+  consol.log('oops'); // eslint-disable-line no-undef
```

Comment:
No reason to cause an implicit reference error here - this can just be `throw new ReferenceError(...)`

Reply:
I did not want to change to much on the code and left the tests pretty much as they were. I will change it though.